resource/css/styles.css

```diff
@@ -2227,6 +2227,29 @@ body, .versal, h1, h2, h3, p, .versal-bold {
   to   { opacity: 1; }
 }
 
+/* CSS custom scrollbars
+ ************************************/
+ :root {
+  --scrollbarWidth: 14px;
+  --scrollbarBg: #b9c1c6;
+  --scrollbarThumb: #474b4f;
+}
```

Comment:
This file already has a :root block elsewhere with colors, font sizes, etc. Maybe worth keeping the root variables all in a single place.

Reply:
Yep. And the variable names should be brought in line with the naming and conventions of the other css variables. To nudge this in the right direction, the beginning of the :root element in styles.css could look like this, with the corresponding changes for the variable names in styles.css row 2237 onwards:

```
:root {
  /* Color definitions */
  --dark-color: #394554; /* topbar nav link text, dropdown text, buttons */
  --secondary-dark-color: #3C535B; /* disabled buttons, bg of "content language" dropdown label in header */
  --medium-color: #00748F; /* used for skosmos service name (top left logo), link texts (a), header bar, hovered dropdown menu items, placeholders in form input fields */
  --light-color: #D4EDEB; /* used for concept property dividers */
  --link-hover-color: #23527C; /* used for hovered links, clipboard icon */
  --tooltip-bg-color: #1E1E1E;
  --tooltip-text-color: #00ACD3;
  --tooltip-border-color: #75889F; /*also used as typeahead focus border */
  --main-bg-color: #D5DBDE;
  --alert-color-pale: #D95F8A;
  --alert-color-bright: #ED0D6C;
  --uri-id-text-color: #006621; /* only used in search results */

  --scroll-box-color: #474b4f; /* used for the sidebar scroll */
  --scroll-bar-color: #b9c1c6; /* used for the sidebar scroll */
  --scroll-bar-width: 14px; /* used for the sidebar scroll */
```